### PR TITLE
Final updates v1.3

### PR DESCRIPTION
**Pull request by janden**
Some final changes before the v1.3 release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ at close-to-FFT speeds;
 under the hood, this involves detailed kernel design, custom spreading/interpolation stages, and plain FFTs performed by cuFFT.
 See the [documentation for FINUFFT][3] for a full mathematical description of the transforms and their applications to signal processing, imaging, and scientific computing.
 
+**Note**: We are currently in the process of adapting the cuFINUFFT interface to closer match that of FINUFFT. This will likely break code depending on the current interface once the next release is published. At this point we will publish a migration guide that will detail the exact changes to the interfaces.
```
Comment (ahbarnett) [inline README.md:22]: I think we should also say that this is the last release of cufinufft as its own repo, and future access to cufinufft will be as part of the main finufft repo.

File: python/cufinufft/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # cuFINUFFT v1.2 Python package
```
Comment (ahbarnett) [inline python/cufinufft/README.md]: 1.3

File: setup.py [hunk 3]
```diff
@@ -29,7 +29,7 @@
 # Python Package Setup
 setup(
     name='cufinufft',
-    version='1.2',
+    version='1.3',
     author='Yu-shuan Melody Shih, Garrett Wright, Joakim Anden, Johannes Blaschke, Alex Barnett',
     author_email='yoyoshih13@gmail.com',
```
Comment (ahbarnett) [inline setup.py:34]: How about your email here, Joakim? :)

**Review by ahbarnett — APPROVED** [review]
This is good, if we add the indication re the shift to finufft repo. (maybe py users don't notice..>) Thanks, Alex

File: README.md (hunk 1 above)
Comment (janden) [reply README.md:22]: Yeah I'll put that in the main readme (this one's only for the Python package).

File: setup.py (hunk 3 above)
Comment (janden) [reply setup.py:34]: Sure why not.

Comment (janden) [reply setup.py]: Done (see 1472cb8c96fd7dba21df25336ba292cf07cbbb6a).

File: README.md (hunk 1 above)
Comment (janden) [reply README.md:24]: Done (see 21cef9d53428b3b7359660d53737fc6881be730a).

**Pull request merged**